tasks/main.yml

```diff
@@ -48,7 +48,7 @@
   rpm_key:
     key: /usr/libexec/dell_dup/{{ item }}
   loop: "{{ dell_dsu_gpg_keys }}"
-  when: dell_dsu_rollout_gpg_keys
+  when: dell_dsu_rollout_gpg_keys and not ansible_check_mode
```

Comment:
Looks cool. I haven't used that variable ansible_check_mode so I'd like to test this a bit before I merge it.

Reply:
sure, docs are here: https://docs.ansible.com/ansible/latest/user_guide/playbooks_checkmode.html#information-about-check-mode-in-variables